gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFormat.java

```diff
@@ -128,14 +122,10 @@ static Path eclipseConfigFile(Project project) {
     private static class PalantirJavaFormatterFunc implements FormatterFunc {
         private static final JavaFormatterOptions OPTIONS =
                 JavaFormatterOptions.builder().style(Style.PALANTIR).build();
-        private final FormatterService formatterService =
-                Iterables.getOnlyElement(ServiceLoader.load(FormatterService.class));
 
         @Override
         public String apply(String input) throws Exception {
-            ImmutableList<Replacement> replacements = formatterService.getFormatReplacements(
-                    OPTIONS, input, ImmutableList.of(Range.closedOpen(0, input.length())));
-            return JavaOutput.applyReplacements(input, replacements);
+            return Formatter.createFormatter(OPTIONS).formatSourceAndFixImports(input);
```

Comment:
Ok so we're just bypassing the `palantir-java-format-api`, but I guess we were already because we used the JavaOutput class.